pkg/minikube/registry/drvs/kvm2/kvm2.go

```diff
@@ -86,7 +90,32 @@ func configure(mc config.MachineConfig) interface{} {
 	}
 }
 
+// check whether current user is in either libvirt or libvirtd group
+func isUserInVirtGroup() (bool, string) {
+	vid, _ := group.LookupGid("libvirt")
+	vidd, _ := group.LookupGid("libvirtd")
+
+	u, _ := user.Current()
+	o, _ := u.GroupIds()
+	uf := false
+	for _, g := range o {
+		i, _ := strconv.Atoi(g)
+		if (i == vid) || (i == vidd) {
+			uf = true
+			break
+		}
+	}
+
+	return uf, u.Name
+}
+
 func status() registry.State {
+	ig, n := isUserInVirtGroup()
+
+	if !ig {
```

Comment:
Let"s do the check after `virsh domcapabilities`.

Reply:
Done